stack.8.10.7.yaml

```diff
@@ -0,0 +1,9 @@
+resolver: lts-18.21
+
+packages:
+- rhine
+- rhine-examples
+- rhine-gloss
+
+extra-deps:
+- time-domain-0.1.0.0@sha256:48146872e85f46399e8b8d7ff7c3d3b7f530e2de167b6893ba3ce649ba2e2e75,800
```

Comment:
I mean, I need to maintain that on stackage. Either way, I guess if I start to maintain it on stackage now, it will not be backported to older `lts`'s, right?

Reply:
Reading the [documentation](https://github.com/commercialhaskell/stackage/blob/master/MAINTAINERS.md) it seems that it is possible to add a package to at least the last LTS, but it might be some work (see "Adding a package to an LTS snapshot"). I don't think it is a big issue that time-domain is in "extra-deps".